src/ert/analysis/_es_update.py

```diff
@@ -43,6 +46,99 @@ class SmootherSnapshot:
     update_step_snapshots: Dict[str, "UpdateSnapshot"] = field(default_factory=dict)
 
 
+def _get_A_matrix(
+    temporary_storage: Dict[str, "npt.NDArray[Any]"],
+    parameters: List[update.Parameter],
+) -> Any:
+    matrices: Any = []
+    for p in parameters:
+        if p.active_list.getMode() == ActiveMode.ALL_ACTIVE:
+            matrices.append(temporary_storage[p.name])
+        elif p.active_list.getMode() == ActiveMode.PARTLY_ACTIVE:
+            matrices.append(
+                np.take(
```

Comment:
Is this the same as `temporary_storage[p.name][p.active_list.get_active_index_list(), :]`?